src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/markup/MarkdownFormatterFactory.java

```diff
@@ -104,4 +104,14 @@ public String format(Text node, FormatterFactory formatterFactory) {
             }
         };
     }
+
+    @Override
+    public Formatter<Link> linkFormatter() {
```

Comment:
Please check if this is covered by the existing Markup formatter tests, or whether a new test needs added explicitly